dspace-api/src/test/java/org/dspace/content/RelationshipMetadataServiceIT.java

```diff
@@ -233,23 +264,52 @@ public void testAuthorDeleteRelationshipCopyToRightItem() throws Exception {
     public void testDeleteAuthorRelationshipCopyToBothItems() throws Exception {
```

Comment:
While this isn't a fault of this PR, I notice this test says it's copying the relationship information to **both** items, but the tests only seem to prove the copy worked to the **leftItem**?   Should we consider correcting this now, so that it also checks that information was copied to the **rightItem**?

Reply:
@benbosman : If you get a chance, could you clarify / answer some of my questions above?  Overall, this PR seems like a good change to get in soon...but, some of the *test* changes are confusing to me.  Thanks.

Reply:
Samuel has added the test for the right item, proving the `relation.*` field is also copied there